src/Nat.mo

```diff
@@ -27,6 +28,9 @@ module {
   /// ```
   public func toText(n : Nat) : Text = Int.toText n;
 
+  /// Computes a hash from the least significant 32-bits of `n`, ignoring other bits.
+  public func hash(n : Nat) : Hash.Hash = Int.hash n;
```

Comment:
why not `let hash : Nat -> Hash.Hash = Int.hash`?

Reply:
eliminates some call overhead

Reply:
Sure.